vsphere/datadog_checks/vsphere/vsphere.py

```diff
@@ -893,8 +893,8 @@ def collect_metrics(self, instance):
                     max_historical_metrics = int(vcenter_settings[0].value)
                 if max_historical_metrics < 0:
                     max_historical_metrics = float('inf')
-            except Exception:
-                pass
+            except Exception as e:
+                self.log.warning("Error getting maxQueryMetrics setting: %s", e)
```

Comment:
This is actually expected in most situations where the config variable is not set.
Let's not make the error message/level more alarming that it should be. Also it could be nice to add the value we default to (DEFAULT_MAX_HIST_METRICS)